aten/CMakeLists.txt

```diff
@@ -43,7 +43,6 @@ IF(MSVC)
   # we want to respect the standard, and we are bored of those **** .
   ADD_DEFINITIONS(-D_CRT_SECURE_NO_DEPRECATE=1)
   LIST(APPEND CUDA_NVCC_FLAGS "-Xcompiler /wd4819 -Xcompiler /wd4503")
-  ADD_DEFINITIONS(-DTH_EXPORTS)
```

Comment:
Is there a reason this has to be in specific subfolders now? I'd expect it to only change the build command if it were set here.